CONTRIBUTING.md

```diff
@@ -32,7 +32,7 @@ $ pip install -e .
 Run tests:
 
 ```sh
-$ python setup.py test
+$ tox --pre
```

Comment:
`python setup.py test` is deprecated:

https://github.com/pypa/setuptools/issues/1684